basic/tailscale_ssh_access/tailscale_ssh_access.tf

```
@@ -1,10 +1,10 @@
 # An AWS Secrets Manager Secret to hold your Tailscale API Key. Set the value with:
-# aws secretsmanager put-secret-value --secret-id "main/tailscale-api-key" --secret-string "YOUR-TAILSCALE-API-KEY"
+# aws secretsmanager put-secret-value --secret-id "${local.environment_name}/tailscale-api-key" --secret-string "YOUR-TAILSCALE-API-KEY"
 resource "aws_secretsmanager_secret" "tailscale_api_key" {
-  name        = "main/tailscale-api-key"
+  name        = "${local.environment_name}/tailscale-api-key"
```

```suggestion
  name        = "sym/${local.environment_name}/tailscale-api-key"
```

Let's prefix the secret here too